apps/docs/pages/guides/database/postgres/roles-superuser.mdx

```diff
@@ -0,0 +1,27 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'roles',
+  title: 'Roles, Superuser Access and Unsupported Operations',
+  slug: 'roles',
+}
+
+Supabase provides the default `postgres` role to all instances deployed. Superuser access is not given as it allows destructive operations to be performed on the database.
+
+To ensure you are not impacted by this, additional privileges are granted to the `postgres` user and we have developed Supautils to provide helper functions for certain privileged operations.
```

Comment:
I don't think we need to mention "Supautils" - users won't know what it is. We can just say abstractly that the `postgres` role is a special role that can perform some superuser-only actions.